backend-gd/config/config.json

```diff
@@ -1,13 +1,13 @@
 {
   "development": {
-    "username": "danielhook",
+    "username": "Jordan",
     "password": null,
     "database": "great-date",
     "host": "127.0.0.1",
     "dialect": "postgres"
   },
   "test": {
-    "username": "danielhook",
+    "username": "Jordan",
```

Comment:
We should add this to gitignore btw, so we don't have to keep switching usernames and shit